elliottlib/errata.py

```diff
@@ -578,6 +578,8 @@ def add_bugzilla_bugs_with_retry(advisory_id: int, bugids: List, noop: bool = Fa
 
     existing_bugs = bzutil.BugzillaBugTracker.advisory_bug_ids(advisory)
     new_bugs = set(bugids) - set(existing_bugs)
+    # check 150 bugs will take around 5 mins, skip check if there are too many bugs
```

Comment:
I think if you skip this check, it will be likely to see an error like "Bug X is already attached to another advisory". I would prefer correctness than performance.